test/Autofac.Test/SourceRegistrationExtensionsTests.cs

```diff
@@ -26,7 +26,7 @@ public void RegisterSource()
                 Assert.True(source.RegistrationsForCalled);
             }
 
-            new ObjectRegistrationSource();
+            _ = new ObjectRegistrationSource();
```

Comment:
Does this line actually do anything? I might be missing it.

Reply:
Yeah, this is weird. I'll remove that.